pkg/webservice/handlers_test.go

```diff
@@ -2602,3 +2607,82 @@ func NewResponseRecorderWithDeadline() *ResponseRecorderWithDeadline {
 		ResponseRecorder: httptest.NewRecorder(),
 	}
 }
+
+func TestCompressGetQueueApplicationAPI(t *testing.T) {
```

Comment:
As we apply the compression to all restful APIs by changing the `webservice` directly, it seems we should add test for `webservice` instead of verifying all APIs here. For example: we can modify `newRouter` to make it accept custom routes, and then we write some dumb routes in testing to check the gzip compression.

Reply:
+1 for the dummy route

Reply:
The test about `webservice` has been added. I don't think we should modify `newRouter` just for the test case, so I kept it as it is. Instead, I simulate the action in `webservice` and then check whether the result is correct.